docs/iceberg-rest-service.md

```diff
@@ -12,62 +12,84 @@ The Apache Gravitino Iceberg REST Server follows the [Apache Iceberg REST API sp
 
 ### Capabilities
 
-- Supports the Apache Iceberg REST API defined in Iceberg 1.3.1, and supports all namespace and table interfaces. `Token`, and `Config` interfaces aren't supported yet.
+- Supports the Apache Iceberg REST API defined in Iceberg 1.5, and supports all namespace and table interfaces. The following interfaces are not implemented yet:
+  - token
+  - view
+  - multi table transaction
+  - pagination
 - Works as a catalog proxy, supporting `Hive` and `JDBC` as catalog backend.
 - Provides a pluggable metrics store interface to store and delete Iceberg metrics.
-- When writing to HDFS, the Gravitino Iceberg REST catalog service can only operate as the specified HDFS user and
-  doesn't support proxying to other HDFS users. See [How to access Apache Hadoop](gravitino-server-config.md#how-to-access-apache-hadoop) for more details.
+- Supports HDFS and S3 storage.
 
 :::info
-Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `1` by default.
 Builds with Hadoop 2.10.x. There may be compatibility issues when accessing Hadoop 3.x clusters.
 :::
 
-## Apache Gravitino Iceberg REST catalog service configuration
-
-Assuming the Gravitino server is deployed in the `GRAVITINO_HOME` directory, you can locate the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md). There are four configuration properties for the Iceberg REST catalog service:
+## Server management
+
+There are three deployment scenarios for Gravitino Iceberg REST server:
+- As a standalone server within the Gravitino Iceberg REST server package.
+- As a standalone server within the Gravitino server package.
+- As an auxiliary service within the Gravitino server.
```

Comment:
A standalone server with the Gravitino Iceberg REST server package.

Reply:
done